overlord/snapstate/snapstate_test.go

```diff
@@ -5718,7 +5740,7 @@ func (s *snapmgrTestSuite) TestInstallMany(c *C) {
 	c.Check(installed, DeepEquals, []string{"one", "two"})
 
 	for _, ts := range tts {
-		verifyInstallUpdateTasks(c, 0, 0, ts, s.state)
+		verifyInstallUpdateTasks(c, install, 0, ts, s.state)
```

Comment:
This method is reading a bit awkward now. verifyInstallUpdate but really install. Let's please split them up:

* verifyInstallTasks
* verifyRefreshTasks